api/tools/src/main/java/org/pmiops/workbench/tools/institutions/LoadInstitutions.java

```diff
@@ -28,6 +30,8 @@
   InstitutionMapperImpl.class,
   InstitutionUserInstructionsMapperImpl.class,
   PublicInstitutionDetailsMapperImpl.class,
+  InstitutionEmailDomainMapperImpl.class,
```

Comment:
FWIW, I'd be fine lumping all the institution mapping functions into one Mapper. It would simplify dependencies, and there's hardly any additional cost to pulling functions you don't need.